trame_client/ui/core.py

```diff
@@ -66,3 +98,36 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, exc_traceback):
         self._original_root.__exit__(exc_type, exc_value, exc_traceback)
         self.flush_content()
+
+    # -------------------------------------------------------------------------
+    # Jupyter display integration
+    # -------------------------------------------------------------------------
+
+    def __jupyter_content(self):
```

Comment:
Let's not name-mangle this and just have one underscore -- for the off chance we want to use this somewhere else:

```suggestion
    def _jupyter_content(self):
```

(needs to be changed where used too)